packages/runtimes/js/src/JSRuntime.js

```diff
@@ -1,8 +1,5 @@
 // @flow
```

Comment:
Calling out `JSRuntime` as an example of a runtime using the api. This one cleaned up quite a bit.